src/transactions/index.js

```diff
@@ -14,6 +14,8 @@ export default function TransactionPage() {
   const handleStopCreating = () => {
     setisCreating(false);
   };
+//   const filters = { categoryId: 0, payee: "Ullrich Group" };
+
```

Comment:
Let's uncomment this so we don't push broken code to the main repository, but lets change it so it's not filtering at all, maybe like this:

const filters = { categoryId: null, payee: null }